docs/tutorials/storage.rst

```diff
@@ -0,0 +1,202 @@
+    :align: center
+
+    Visualizing Images
+
+
+Visualizing a FITS file with custom serialization
+-------------------------------------------------
+
+By default, DeepForge uses pickle to serialize `artifacts` and load them on operations. In this example, we will look into how to visualize fits images using a custom serialization in DeepForge. The FITS files we will use are available to download `here <http://www.astropy.org/astropy-data/visualization/reprojected_sdss_r.fits.bz2>`_, `here <http://www.astropy.org/astropy-data/visualization/reprojected_sdss_g.fits.bz2>`_ and `here <http://www.astropy.org/astropy-data/visualization/reprojected_sdss_i.fits.bz2>`_. Before we start uploading the FITS files, we have to write a custom serializer for the FITS file, which can then be utilized for consumption in our operation.
+
+For visualizing FITS files, we will be using the `astropy <https://github.com/astropy/astropy>`_ python library. Function ``astropy.io.fits.open``, is able to handle a raw compressed binary FITS file or a simple filename. To visualize the file in DeepForge you can register a serializer for FITS file by writing a new class for handling them. For that, navigate to `My Utilities` and add a new python module called ``fits_file.py`` as highlighted in the figure below:
+
+.. figure:: custom_utils.png
+    :align: center
+
+    Custom Utils
+
+After you click on Enter, a text editor will open. Copy and paste the following content:
+
+.. code-block:: python
+
+    from astropy.io import fits
+
+    class FitsFile():
+
+        def __init__(self, file):
+            self.name = file.name
+            self.file = file
+
+        def load(self):
+            return fits.open(self.file)
+
+        def save(self, outfile):
+            os.system('ln -s ' + self.name + ' ' + outfile.name)
+
+    def load(infile):
+        fits_file = FitsFile(infile)
+        return fits_file.load()
+
+    def save(fits_file, out_file):
+        fits_file.save(out_file)
+
+Figure below shows the text editor with the content:
+
+.. figure:: fits_serializer.png
+    :align: center
+
+    FITS Serializer
+
+Now, we are ready to import files as described in :ref:`previous section <Loading a pickled file>`, but this time specifying the `Data Type` as ``utils.fits_file.FitsFile``. This is shown in the figure below (For one FITS File):
+
+.. figure:: custom_serialized_artifact_upload.png
+    :align: center
+
+    Uploading Artifact with a Custom Serializer
+
+Now we are ready to use the files. Follow similar steps for all the FITS files you want to use. In our case, we want to visualize them using ``make_lupton_rgb`` function in ``astropy.visualization`` package. The pipeline `Visualize FITS` is shown below:
+
+.. figure:: visualize_fits.png
+    :align: center
+
+    Pipeline Visualize FITS
+
+The `pipeline` has a single operation called `VisualizeFITS` that uses the FITS file we uploaded with custom serializer.
+
+Using data available different storage backends
```

Comment:
This doesn't really make sense. I suspect you are referring to importing existing data from a storage backend...?